actionview/lib/action_view/renderer/partial_renderer/collection_caching.rb

```diff
@@ -0,0 +1,74 @@
+require 'active_support/core_ext/object/try'
+
+module ActionView
+  module CollectionCaching
+    extend ActiveSupport::Concern
+
+    included do
+      # Use Rails.cache or the fallback cache store if Action View is used without Rails.
+      mattr_accessor(:collection_cache) do
+        Rails.try(:cache) || ActiveSupport::Cache::MemoryStore.new
```

Comment:
Maybe this could be set in railtie? Not sure if there is a `Rails` defined if `actionview` is used standalone.